app/lang/fr/setup.php

```diff
@@ -5,7 +5,7 @@
     'title'             => 'Installer Cachet',
     'service_details'   => 'Details du service',
     'status_page_setup' => 'Installation de la page de statut',
-    'show_support'      => 'Show support for Cachet? <small>Places a link in the footer linking to Cachet.</small>',
+    'show_support'      => 'Montrer votre support pour Cachet? <small>Affiche un lien vers Cachet dans le pied de page.</small>',
     'admin_account'     => 'Compte administrateur',
     'complete_setup'    => 'Terminer l\'installation',
     'completed'         => 'Cachet a été configuré avec succès !',
```

Comment:
Hmmm, it seems to be a stylistic choice. It does seem to be preferred in that language. I think we shouldn't do it though. It spoils the consistency.